src/Catalog/NuGet.Services.Metadata.Catalog.csproj

```diff
@@ -266,7 +266,7 @@
       <Version>1.0.8.3533</Version>
     </PackageReference>
     <PackageReference Include="NuGet.Services.Entities">
-      <Version>4.4.5-master-2699519</Version>
+      <Version>4.4.5-dev-2939401</Version>
```

Comment:
Today, the later version of NuGetGallery.Core brings this stuff directly to the NuGet.Services.AzureSearch project. We should take that path to ensure a) these older entry points remain unaffected and b) the rest of the NuGetGallery.Core code that uses NuGet.Services.Entities is consistent. In short, NuGet.Services.AzureSearch has a NuGetGallery.Core older than this NuGet.Services.Entities. This should not happen.

Reply:
To be clear this means updating NuGetGallery.Core in jobs and pulling it in through NuGet.Services.AzureSearch's Validation.Common.Job dependency.

Reply:
Actually I think the NuGet.Jobs update will get done with https://github.com/NuGet/NuGet.Jobs/pull/796, if you want me to push that through first.

Reply:
I merged it. You should be able to use Validation.Common.Job from this build:
https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=2942767

Reply:
I was planning on updating these versions when a master version of `NuGet.Services.Entities` was released, but you're saying I need to update the `NuGetGallery.Core` references as well?

Reply:
Okay will do!

Reply:
Since NuGetGallery.Core is used in this repository by Azure Search, it is not guaranteed that the entities changes you pull into the graph are compatible with NuGetGallery.Core. Perhaps these changes are... but it's not worth threading the needle like that when you can move them both forward.

Reply:
We don't need this change now.

Reply:
Moved the `NuGetGallery.Core` version in `NuGet.Services.AzureSearch` forward. Is that all you needed me to do?

To be honest, I think we should maybe consider having some root project with like these so we can push versions of dependencies forward without worrying about conflicts like these.